internal/uow/exec_test.go

```
@@ -0,0 +1,764 @@
+				return &transaction{
+					id:     txID,
+					status: txStatusInProgress,
+					requests: map[storage.Driver]*storage.Request{
+						driverExecError: {
+							Val:  "INSERT INTO users.users (user_id) VALUES ($1)",
+							Args: []any{"1"},
+						},
+					},
+					failedDriver: "",
+				}
+			}(),
```

golangci-lint (2.5.0)

[error] missing whitespace above this line (too many lines above return)

(wsl_v5)